### ISSUE TITLE
add support for favicon

### ISSUE THREAD

**Issue by tomachalek** [czcorpus/wdglance#335]
Configurable via one of json config files

**Comment by tomachalek** [comment]
The configuration should be probably in `server.json` as the 1st level key:
```json
{
  "favicon": {
    "contentType": "image/png",
    "path": "absolute/path/to/favicon/file.png"
  }
}
```
This should be transformed into something like 
```html
<link rel="icon" type="image/png" href="absolute/path/to/favicon/file.png" />
```

**Comment by tomachalek** [comment]
Fixed in c9be48adb7b2246d51b643569b4779a346eb110c
